src/config.rs

```diff
@@ -0,0 +1,52 @@
+use clap::Parser;
+use derive_builder::Builder;
+use tree_sitter_grep::SupportedLanguage;
+
+use crate::{
+    rule::{ResolvedRule, Rule},
+    rules::{no_default_default_rule, no_lazy_static_rule, prefer_impl_param_rule},
+};
+
+#[derive(Builder, Parser)]
+#[builder(setter(strip_option, into))]
+pub struct Config {
+    #[arg(short, long, value_enum)]
+    pub language: SupportedLanguage,
+
+    #[arg(long)]
+    #[builder(default)]
+    pub rule: Option<String>,
+
+    #[arg(long)]
+    #[builder(default)]
+    pub fix: bool,
+
+    #[arg(long)]
+    #[builder(default)]
```

Comment:
Is it possible to turn this on by default? (I might be getting mixed up with how I have mostly used the `builder` macro through my own macro, so have a bit more control.)